### DISCUSSION TITLE
Decluttering printed information (Don't use 0 unless it's the default. and others)

### DISCUSSION THREAD

**Discussion by jtagcat** [archlinux/archinstall#331] [Ideas]
![image](https://user-images.githubusercontent.com/38327267/115570820-f47ff300-a2ad-11eb-97ab-4a18e400060e.png)

For me, 0 implies, that it's the default (can be selected by just pressing enter).  
Don't show 0 at all, or move the 'if you don't want to install one' to `0: Don't use a preconfigured profile`.

Thoughts?

**Comment by Torxed** [comment]
Most if not all steps are optional and can be skipped with just hitting Enter.
The text `Leave blank and hit enter to skip this step` should also indicate this.

I'm moving this to a conversation topic as it's not an issue in of itself in my opinion.
Perhaps a healthy discussion can hash out a suggestion on how to better put emphasis on the skipping part.

**Reply by jtagcat** [reply]
The problem with `Leave blank and hit enter to skip this step` is that it's long and wordy to add to each prompt. For example Printing `optional means it can be skipped` at the beginning, and using `(optional):` is more visually distinct.

I find, that the current prompts are not 'tabav' (on-point, exact, brief, yet at the same time giving good information).  
As an user, I want to quickly gasp what is wanted of me. In the screenshot above, it is already said twice, that the field is optional.

**Reply by SecondThundeR** [reply]
You are right about these moments. In general, it seems to me that the concept of installation steps and selection of options should be slightly improved in the future, as user experience should be a top priority in future releases.

At the moment, the current problems are that it's necessary to add some separators for each stage of the installation so that it doesn't turn into a giant piece of text, and it's necessary to somehow rethink the output of the options so that the first item doesn't have a zero index and make the description of the current stage more understandable for users.

If you don't mind helping with this and contributing, then we look forward to your suggestions and improvements!

**Comment by jtagcat** [comment]
I think it might get controversial, but thoughts about clearing the screen when proceeding to the next step?

Adding whitespace or =======HEADER======== to differentiate isn't as nice.

I didn't even think of this, that when proceeding the installation, 10 lines get printed, but you don't have a clear indicator on where to continue reading (as in the user might accidentally start reading the last part again).

**Comment by Torxed** [comment]
Would having a more traditional "arch outpu" (think pacman) solve parts of this issue? https://github.com/archlinux/archinstall/issues/220

```
 Status message!
> Other message!
:: Action message!
Error: Error message!
==> Action Warn message!
==> Action Info message!
Success: Success message!
Warning: Warning message!
 -> Notice message!
(1/5) Progress message!
```
